### PR TITLE
Add `Post.campaign`, `reviewPost`, and `tagPost`.

### PR DESCRIPTION
**Pull request by DFurnes**
This pull request adds some features for displaying & reviewing posts in Rogue:

🍡 Adds a `campaign: Campaign` field to the `Post` type. bd83d3f

👁‍🗨 Adds a `reviewPost` mutation to review a given post. 5cc884d

🏷 Adds a `tagPost` mutation to add/remove tags on a given post. afac6af

References [#169512736](https://www.pivotaltracker.com/story/show/169512736).

### CODE REVIEW

File: src/schema/rogue.js [hunk 1]
```diff
@@ -381,6 +385,20 @@ const typeDefs = gql`
...
+      id: Int!
+      "The status to give this post."
+      status: ReviewStatus!
+    ): Post
+    "Add or remove a tag on a post. Requires staff/admin role."
+    tagPost(
+      "The post ID to review."
+      id: Int!
+      "The tag to add or remove on this post."
+      tag: String!
```
[the remark this answers is not in the document]
Comment (DFurnes) [reply src/schema/rogue.js]: I'd like to, but we don't currently have that in the schema! Do you think that makes sense to do now?

[the remark this answers is not in the document]
Comment (DFurnes) [reply src/schema/rogue.js]: I'm leaning towards "yes", I think...

[the remark this answers is not in the document]
Comment (katiecrane) [reply src/schema/rogue.js]: I think yes but up to you! Too out of scope?

[the remark this answers is not in the document]
Comment (DFurnes) [reply src/schema/rogue.js]: Oh, although now I'm seeing a bunch of places where we query this as a string so perhaps not so easy. 😖 I'll make a ticket and we can take care of it on it's own later on.

[the remark this answers is not in the document]
Comment (DFurnes) [reply src/schema/rogue.js]: Created [#169549410](https://www.pivotaltracker.com/story/show/169549410).

[the remark this answers is not in the document]
Comment (katiecrane) [reply src/schema/rogue.js]: Perfect!!

**Pull request merged**